### PR TITLE
feat(dvcx-worker): Add DVCx Worker definition

### PR DESCRIPTION
**Pull request by mjasion**
This commit adds DVCx Worker deployment, 
with ephemeral-storage configuration, which allows for configuring:
* emptyDir
* pvcTemplate

Additionally:
* ~Temporary the base Studio worker is scaled to 0.~ 
* Added more documentation  to `studioWorker`
* Removed `studio:` prefixes from some docs

Closes https://github.com/iterative/itops/issues/2181

### CODE REVIEW

File: charts/studio/templates/deployment-studio-worker.yaml [hunk 1]
```diff
@@ -5,9 +5,10 @@ metadata:
   labels:
     {{- include "studio-worker.labels" . | nindent 4 }}
 spec:
-  {{- if not .Values.studioWorker.autoscaling.enabled }}
-  replicas: {{ .Values.studioWorker.replicaCount }}
-  {{- end }}
+{{/*  {{- if not .Values.studioWorker.autoscaling.enabled }}*/}}
+{{/*  replicas: {{ .Values.studioWorker.replicaCount }}*/}}
+{{/*  {{- end }}*/}}
+  replicas: 0
```
Comment (mjasion) [inline charts/studio/templates/deployment-studio-worker.yaml:11]: When we will have splited images, this has to be reverted. @iterative/platform

File: charts/studio/values.yaml [hunk 2]
```diff
@@ -206,6 +206,8 @@ nginx:
   - name: blobvault
     mountPath: /blobvault
 
+  # -- Nginx for blobvault configuration
+  # @default -- see in `values.yaml`
```
Comment (mjasion) [inline charts/studio/values.yaml:210]: Added this, because README.md was very big...

File: charts/studio/values.yaml [hunk 3]
```diff
@@ -551,10 +566,96 @@ studioWorker:
...
+  # -- Worker affinity
+  affinity: {}
+
+# -- Studio DVCx Worker settings group
+studioDvcxWorker:
+
+  # -- DVCx worker image settings
+  image:
+    # -- DVCx worker image repository
+    repository: docker.iterative.ai/studio-backend
```
Comment (mjasion) [inline charts/studio/values.yaml:582]: This also needs to be changed when we will have separate images for worker @iterative/platform

File: .github/workflows/helm-lint-and-install.yaml [hunk 4]
```diff
@@ -1,4 +1,4 @@
-name: Lint and Test Charts
+helm-lint-and-test: Lint and Test Charts
```
Comment (mjasion) [inline .github/workflows/helm-lint-and-install.yaml:1]:
```suggestion
name: Lint and Test Charts
```

**Comment by jesper7** [comment]
The failing lint test seems like a false positive. I couldn't reproduce it in manual testing, so let's merge this

**Review by jesper7 — APPROVED** [review]

**Review by 0x2b3bfa0 — APPROVED** [review]
It passes tests 🤷🏼‍♂️

**Pull request merged**